api/objects.proto

```diff
@@ -39,7 +39,9 @@ message Node {
 	// Status provides the current status of the node, as seen by the manager.
 	NodeStatus status = 5 [(gogoproto.nullable) = false];
 
-	Manager manager = 6;
+	// Status of the manager. If the node is not a manager, this field will not
+	// be set.
+	ManagerStatus manager_status = 6;
```

Comment:
I guess this is okay. I kind of liked the name `Manager` because it seemed relatively clean to check whether a node is a manager or not by checking if `node.Manager` exists. When it's called `ManagerStatus`, it might seem more ambiguous, because if `ManagerStatus` is nil that might be interpreted as the status being unavailable.